static/src/js/builder/codelistbuilder.jsx

```diff
@@ -189,16 +189,12 @@ class CodelistBuilder extends React.Component {
             </dl>
             <hr />
 
-            {this.props.searches.length > 0 && (
-              <>
-                <h6>Versions</h6>
-                <ul className="pl-3">
-                  {this.props.versions.map((version) => (
-                    <Version key={version.hash} version={version} />
-                  ))}
-                </ul>
-              </>
-            )}
+            <h6>Versions</h6>
+            <ul className="pl-3">
+              {this.props.versions.map((version) => (
+                <Version key={version.hash} version={version} />
+              ))}
+            </ul>
```

Comment:
Does this match what we do on the non-builder (need a better name for that...) page?  That is, if there's only one version, do we show a list of versions?

Reply:
yes, we do